experimental/generation/generator/packages/library/templates/standard/build.cmd.lg

```diff
@@ -45,8 +45,8 @@ ${join(foreach(locales, locale, copyForLocale(locale)) , EOL())}
 
 # copyForLocale(locale)
 -```
-call copy .\cross-trained\\${prefix}.${locale}.lu .\\${locale}\ /y
-call copy .\cross-trained\\${prefix}.${locale}.qna .\\${locale}\ /y
+call copy .\cross-trained\\${prefix}.${locale}.lu .\language-understanding\\${locale}\ /y
+call copy .\cross-trained\\${prefix}.${locale}.qna .\knowledge-base\\${locale}\ /y
 ```
```

Comment:
This would be better in the template filename rather than relying on this script which will not be run by composer.